indy_client/test/cli/helper.py

```diff
@@ -83,8 +83,14 @@ def ensureConnectedToTestEnv(be, do, cli):
     be(cli)
     if not cli._isConnectedToAnyEnv():
         timeout = waits.expectedClientToPoolConnectionTimeout(len(cli.nodeReg))
-        do('connect test', within=timeout,
-           expect=['Connected to test'])
+        connect_and_check_output(do, cli.txn_dir, timeout)
+
+
+def connect_and_check_output(do, netwotk, timeout=3, expect=None, mapper=None):
+    if expect is None:
```

Comment:
As I can see, Network is taken from `cli.txn_dir` which is set by `sub-directory` parameter in constructor, and it can be None. How do we make sure that we connect to test?

Reply:
connect_and_check_output is used only with named clis, so this is ok for now